bigframes/ml/core.py

```diff
@@ -326,7 +326,7 @@ def create_model(
         if y_train is None:
             input_data = X_train.cache()
         else:
-            input_data = X_train.cache().join(y_train.cache(), how="outer")
+            input_data = X_train.join(y_train.cache(), how="outer").cache()
```

Comment:
this should be y_train without cache() as well?

Reply:
yes, fixed